hazelcast-hibernate53/src/main/java/com/hazelcast/hibernate/HazelcastLocalCacheRegionFactory.java

```diff
@@ -68,7 +68,9 @@ protected RegionCache createTimestampsRegionCache(final String unqualifiedRegion
                 sessionFactory.getSessionFactoryOptions()
         );
 
-        return new TimestampsRegionCache(this, qualifiedRegionName, instance);
+        TimestampsRegionCache timestampsRegionCache = new TimestampsRegionCache(this, qualifiedRegionName, instance);
+        cleanupService.registerCache(timestampsRegionCache);
```

Comment:
I think we should not register any clean up procedure for timestamps cache. I see you registered here not to make an eviction but for unsubscribing topic on destroy. However, if it evicts a timestamp entity this might invalidate many other valid entities in other regions.

Reply:
We can have two types of registrations. Shutdown registrations and cleanup job registrations. What do you think?
Also, TimestampRegionCache cleanup method is empty, so its not doing evictions.

Reply:
> Also, TimestampRegionCache cleanup method is empty, so its not doing evictions.

Sorry, I missed that point. It's OK then.